oci-unit-tests/alertmanager_test.sh

```diff
@@ -14,7 +14,11 @@
 # The name of the temporary docker network we will create for the
 # tests.
 readonly DOCKER_PREFIX=oci_alertmanager_test
-readonly DOCKER_IMAGE="${DOCKER_IMAGE:-ubuntu/prometheus-alertmanager:edge}"
+readonly DOCKER_REGISTRY="${DOCKER_REGISTRY:-docker.io}"
+readonly DOCKER_NAMESPACE="${DOCKER_NAMESPACE:-ubuntu}"
+readonly DOCKER_PACKAGE="${DOCKER_PACKAGE:-prometheus-alertmanager}"
+readonly DOCKER_TAG="${DOCKER_TAG:-edge}"
+readonly DOCKER_IMAGE="$DOCKER_REGISTRY/$DOCKER_NAMESPACE/$DOCKER_PACKAGE:$DOCKER_TAG"
```

Comment:
Let's stick to using `${VARIABLE}` instead of `$VARIABLE`.

Reply:
Not that I'm against it, but why? We use the `$VARIABLE` syntax very often across these scripts, and I''m not aware of any recommendation about always using `${VARIABLE}`.

Reply:
You're right that we also use `$VARIABLE` across the scripts.  My comment was more of a personal preference when we use environment variables as part of path components; I think it's more readable this way, TBH.